muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/communicator/body/application/new-message.tsx

```diff
@@ -137,28 +138,40 @@ class CommunicatorNewMessage extends SessionStateComponent<CommunicatorNewMessag
     let content = (closeDialog: ()=>any) => [
       (<InputContactsAutofill modifier="new-messsage" key="1" hasGroupPermission placeholder={this.props.i18n.text.get('plugin.communicator.createmessage.title.recipients')}
         selectedItems={this.state.selectedItems} onChange={this.setSelectedItems} autofocus={!this.props.initialSelectedItems}></InputContactsAutofill>),
-      (<input key="2" type="text" className="form-field form-field--communicator-new-message-subject"
-        placeholder={this.props.i18n.text.get('plugin.communicator.createmessage.title.subject')}
-        value={this.state.subject} onChange={this.onSubjectChange} autoFocus={!!this.props.initialSelectedItems}/>),
-      (<CKEditor key="3" width="100%" height="grow" configuration={ckEditorConfig} extraPlugins={extraPlugins}
-       onChange={this.onCKEditorChange}>{this.state.text}</CKEditor>),
+      (
+       <div className="container container--communicator-subject">
+        <div className="environment-dialog__form-element--wrapper">  
```

Comment:
Where is the BEM block element, it seems that there is only BEM element "environment-dialog__form-element--wrapper" directly here.

If we use modifiers we need to include the base element class name too as modifier is an extra class name and it cannot be the only class name of the html element.